AppCenterIdentity/AppCenterIdentity/Internals/MSIdentityAppDelegate.m

```diff
@@ -0,0 +1,49 @@
+#import "MSIdentityAppDelegate.h"
```

Comment:
Sort

Reply:
Already sorted; we put the header file corresponding to the implementation at the top, and sort the rest alphabetically.

Reply:
Is this the formatter doing this? I've never noticed it :clown_face:.

Reply:
It's a convention that's quite common. I don't think we follow it everywhere.

Reply:
Formatter does this, I believe

Reply:
If the forwarder does this I am fine otherwise I don't think we should do it. I saying this particularly for myself because I am sure I'll forget to do it if not done by the formatter.